payload_queue.go

```diff
@@ -1,86 +1,153 @@
 package sctp
 
 import (
+	"fmt"
 	"sort"
 )
 
-type payloadDataArray []*chunkPayloadData
+////////////////////////////////////////////////////////////////////////////////
+// Pending Queue
 
-func (s payloadDataArray) search(tsn uint32) (*chunkPayloadData, bool) {
-	i := sort.Search(len(s), func(i int) bool {
-		return sna32GTE(s[i].tsn, tsn)
-	})
+type pendingQueue struct {
```

Comment:
Maybe just move this to it's own file instead?

Reply:
Agreed and fixed in 472b620